.github/workflows/functional-tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     secrets: inherit
     strategy:
       matrix:
-        aws-region: ['us-east-1', 'eu-west-1', 'eu-west-2', 'eu-central-1']
```

Comment:
Do we really want to remove `eu-west-1` ?

Reply:
yup, see description, we use us-east-1 as a proxy for what happen in eu-west-1, and keep eu-central-1. But ideally we can keep eu-west-1 if we are in the budget, @mattiamatrix I leave this decision to you :P

Reply:
I think it makes sense, we can add them again if we realize something is missing.